### DISCUSSION TITLE
Broken CSS on Vercel docs website

### DISCUSSION THREAD

**Discussion by tebuevd** [vercel/community#3076] [Bug Report]
### Page to Investigate

https://vercel.com/docs/concepts/monorepos/turborepo#using-turbo-ignore

### Steps to Reproduce

Go to the url https://vercel.com/docs/concepts/monorepos/turborepo#using-turbo-ignore, you should see the following:

<img width="848" alt="image" src="https://github.com/vercel/community/assets/3241533/894567d8-2511-4d3b-b087-69351df9373f">

Tested with Arc, Firefox, Safari

**Comment by raiadi2008** [comment]
i guess something is wrong. /guides urls are not even opening

**Comment by amyegan** [comment]
Thanks for reporting this. Everything seems to be working normally again 😄

**Comment by tebuevd** [comment]
Closing as it's been fixed.
